### PR TITLE
Standardize the error message

### PR DESCRIPTION
**Pull request by daliwei**
cc @pierre for review. This is to standardize the format of the error message.

### CODE REVIEW

**Comment by sbrossie** [comment]
👍

File: src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java [hunk 1]
```diff
@@ -436,4 +439,13 @@ private static void computeLengthArray(final String className, final int[] dotAr
         final Iterable<PluginProperty> propertiesWithDefaults = PluginProperties.merge(defaultProperties, originalProperties);
         return ImmutableList.<PluginProperty>copyOf(propertiesWithDefaults);
     }
+
+    @VisibleForTesting
+    static String formatErrorMessage(final String gatewayError) {
+        if(Strings.isNullOrEmpty(gatewayError)) {
+            return gatewayError;
+        }
+        //Remove any extra spaces between word and convert all to lower-case
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java]: What's the reasoning behind the change? It's going to be weird for messages like `"HTTP response '503: Service Unavailable'` or `Connection reset`.

Also, most messages seem pretty:

* `validation 105 Invalid paRes from issuer`
* `validation 137 Invalid amount specified`
* `validation 126 Bank Account or Bank Location Id not valid or missing`

Comment (daliwei) [reply src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java]: The error messages returned from Adyen sometimes is formatted inconsistently, for example, we see both `Expired Card` and `Expired card`. The upstream service relies on these messages to further aggregated the cause into different categories.

**Comment by pierre** [comment]
Summary of offline discussion:

* Upstream service should use the [killbill-payment-retries-plugin](https://github.com/killbill/killbill-payment-retries-plugin) instead, to bucketize errors.
* The bucketization is today incomplete unfortunately, because Adyen isn't consistent regarding processor response codes returned. We are currently investigating how to fix this.

**Pull request closed without merging**